superset/tasks/scheduler.py

```diff
@@ -47,9 +48,15 @@ def scheduler() -> None:
         return
     with session_scope(nullpool=True) as session:
         active_schedules = ReportScheduleDAO.find_active(session)
+        triggered_at = (
+            datetime.fromisoformat(scheduler.request.expires)
+            - app.config["CELERY_BEAT_SCHEDULER_EXPIRES"]
```

Comment:
not your fault, but having to do it like this just makes me cringe.. 😄 I'm hoping someone can add `request.scheduled_at` to Celery 😉